### ISSUE TITLE
Merge with https://www.w3.org/wiki/WebAccessControl

### ISSUE THREAD

**Issue by michielbdejong** [solid/web-access-control-spec#51]
As @namedgraph pointed out, this is not the only WAC spec.

Does anybody know how and why this situation arose? 

https://www.w3.org/wiki/index.php?title=WebAccessControl&action=history shows edits from  @bblfish, @timbl and others over the period 2009-2018.

https://github.com/solid/web-access-control-spec/commits/master?after=6e7bc5130e47699618c807344b226719bf53db66+34 shows edits from @dmitrizagidulin, @deiu, @timbl, @kjetilk, @TallTed, @acoburn and @elf-pavlik over the period 2016-2019

**Comment by bblfish** [comment]
The wiki was where ideas that were discussed by Dan Connolly and TimBL and others MIT folks on IRC where put together initially. Then things were consolidated on the github repository when solid came to be named.

**Comment by michielbdejong** [comment]
OK! @dmitrizagidulin in 2016 [you created](https://github.com/solid/web-access-control-spec/commit/ff6e9ba60f3bf9d9be9ec789ed837c6dade48140) this document and stated it was a Solid-specific subset of WAC. Was it your intention that https://www.w3.org/wiki/WebAccessControl would live on independently? Or was it your intention to replace that wiki page?

And having come to the current point, what do you think should happen now with that wiki page?

**Comment by namedgraph** [comment]
The WAC ontology has been under the W3C namespace forever. I think what would make most sense is to move both the documentation and the RDF file to a repository under @w3c GitHub account, so people could suggest/review/discuss changes and fixes using the git process. URL redirection should be fairly easy to setup.
Last I knew, @timbl was in control of the vocabulary file.

**Comment by dmitrizagidulin** [comment]
@michielbdejong Yes, the idea was that this repo and the w3c wiki would live on independently. The wiki, and the way Solid was using and extending WAC had diverged before I joined the project. Devs were complaining that there wasn’t a spec for it, so I put together this repo. 

And yes, there are 3 different flavors of WAC out there (4 actually, since the OpenLink crew has their own extensions and their own spec) - this repo, the wiki, and the w3c draft, and open link spec).

**Comment by dmitrizagidulin** [comment]
@michielbdejong 
> And having come to the current point, what do you think should happen now with that wiki page?

No opinion about what should happen to the wiki page; as far as I know, we don’t really have jurisdiction over it, it’s not in our project’s scope.

**Comment by namedgraph** [comment]
@dmitrizagidulin the version that matters most is the one published under http://www.w3.org/ns/auth/acl

**Comment by michielbdejong** [comment]
Got it! Thanks for explaining. Then we should update https://github.com/solid/solid-spec#authorization-and-access-control to say we use "Solid-flavoured WAC" and that it's only one of the four flavours out there.

This spec should also have a preamble explaining that it's only one of four flavours.

**Comment by akuckartz** [comment]
Is it really necessary to have four different "flavors" instead of one standard?

**Comment by michielbdejong** [comment]
I think the other three are mainly unused, but it's not up to us to deprecate them, so we just mention that those alternative versions exist, and we ignore them.

**Comment by bblfish** [comment]
The wiki page being open to anyone to edit, you can add a link from the wiki to the page here.

**Comment by michielbdejong** [comment]
I seem to have an account on https://www.w3.org/community/solid/wiki/ but not on https://www.w3.org/wiki/. I created an account now but it's awaiting review. I'll add the note once my account is active, unless someone else beats me to it! :)

**Comment by namedgraph** [comment]
@michielbdejong I still don't get what Solid is doing. If you're using a "Solid-flavored WAC", then the namespace of such ontology should be distinct from http://www.w3.org/ns/auth/acl.

**Comment by csarven** [comment]
Instead of investing further energy on the differences, can we consolidate the flavours? Bring in the good parts from all to the table. Any particular technical or social barriers?

If trustedWhatever is a shared notion, it makes sense to agree on that even if not all implementations of the spec use it. Aside: it may also mean that systems can function without trustedWhatever in place.

W3C wiki/WAC documents the wider design whereas auth/acl is just a vocab.

We all aim for a single WAC and not further fragment. It is completely okay that there are different flavours now, it only reflects the implementations/experiences from different perspectives. Let's take advantage of that knowledge.

**Comment by kidehen** [comment]
> Instead of investing further energy on the differences, can we consolidate the flavours? Bring in the good parts from all to the table. Any particular technical or social barriers?
> 
> If trustedWhatever is a shared notion, it makes sense to agree on that even if not all implementations of the spec use it. Aside: it may also mean that systems can function without trustedWhatever in place.
> 
> W3C wiki/WAC documents the wider design whereas auth/acl is just a vocab.
> 
> We all aim for a single WAC and not further fragment. It is completely okay that there are different flavours now, it only reflects the implementations/experiences from different perspectives. Let's take advantage of that knowledge.

Yes, but we should be clear about what "consolidation of flavors" entails.  For starters, how is Solid using `http://www.w3.org/ns/auth/acl#`, distinct from its existing specification?
